demos/src/accordion.mustache

```diff
@@ -0,0 +1,38 @@
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="3" data-o-expander-count-selector="li">
+	<h2>Simple List</h2>
+	<ul class="o-expander__content">
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+		<li>item</li>
+	</ul>
+	<a href='#' class="o-expander__toggle o--if-js"></a>
+</div>
+
+<div data-o-component="o-expander" class="o-expander items" data-o-expander-shrink-to="2" data-o-expander-count-selector="li" data-o-expander-accordion="true">
```

Comment:
You are right! Also, this current behaviour could be done by several `o-expander`s concatenated. I've implemented that behaviour, i'll push it when I finish the other changes :)